src/azure-cli/install.sh

```diff
@@ -17,8 +17,8 @@ AZ_EXTENSIONS=${EXTENSIONS}
 AZ_INSTALLBICEP=${INSTALLBICEP:-false}
 INSTALL_USING_PYTHON=${INSTALL_USING_PYTHON:-true}
```

Comment:
Can we bump the patch version for this Feature?

Reply:
done